lib/content_security_policy/default.rb

```diff
@@ -25,16 +25,18 @@ def initialize
       ['/brotli_asset/',       true, true],
       ['/extra-locales/',      false, false],
       ['/highlight-js/',       false, true],
-      ['/javascripts/',        false, true],
+      ['/javascripts/',        false, :subfolder],
       ['/plugins/',            false, true],
       ['/theme-javascripts/',  false, true],
-      ['/svg-sprite/',         false, true],
+      ['/svg-sprite/',         false, :subfolder],
     ]
 
     def script_assets(base = base_url, s3_cdn = GlobalSetting.s3_cdn_url, cdn = GlobalSetting.cdn_url)
       SCRIPT_ASSET_DIRECTORIES.map do |dir, can_use_s3_cdn, can_use_cdn|
         if can_use_s3_cdn && s3_cdn
           s3_cdn + dir
+        elsif (can_use_cdn == :subfolder) && cdn
```

Comment:
Although ruby lets you do this with duck typing, I am not a fan of having a variable use multiple types. I would prefer them all by symbols than some be booleans and some symbols.